_maps/RandomZLevels/modular_maps/caves/caves_lvl2bridge1_broken.dmm

```diff
@@ -0,0 +1,84 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"f" = (
+/turf/template_noop,
+/area/template_noop)
+"l" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/turf_decal/stripes/end,
+/obj/modular_map_connector,
+/obj/structure/railing{
+	dir = 4
+	},
+/obj/structure/railing{
+	dir = 8
+	},
+/turf/open/floor/engine{
+	initial_gas_mix = "LAVALAND_ATMOS"
+	},
```

Comment:
Make this a subtype (and any other instances where you manually edit initial_gas_mix) because otherwise some new mapper will pick it up not understanding that it's been varedited and cause fifty jillion Active Turfs